### PR TITLE
feat: initial support of StoneDB

### PR DESCRIPTION
**Pull request by ZhengLin-Li**
feat: initial support of StoneDB

### CODE REVIEW

File: src/sqlancer/stonedb/gen/StoneDBTableGenerator.java [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+            sb.append("MEDIUMBLOB");
+            break;
+        case LONGBLOB:
+            sb.append("LONGBLOB");
+            break;
+        }
+    }
+
+    public static void optionallyAddPrecisionAndScale(StringBuilder sb) {
+        if (Randomly.getBoolean() && !MySQLBugs.bug99183) {
```
Comment (mrigger) [inline src/sqlancer/stonedb/gen/StoneDBTableGenerator.java:340]: This seems MySQL-specific?

Comment (bajinsheng) [reply src/sqlancer/stonedb/gen/StoneDBTableGenerator.java]: Solved

**Comment by bajinsheng** [comment]
Since some necessary CI failed, I cannot merge the PR now.  @ZhengLin-Li Can you help to fix the CI error of MariaDB, so that I can merge your PR? You can simply add the string "Specified key was too long" to the expected error of MariaDB.

**Comment by bajinsheng** [comment]
@ZhengLin-Li Can you update this branch? @mrigger Can you help to merge this PR? so that Zhenglin can continue.

**Comment by ZhengLin-Li** [comment]
updated

**Comment by bajinsheng** [comment]
I meant updating to the latest commit of the official repo, so that we can merge this pr without conflict. Not updating all your newly written code. And also, don't mix different features in one pr.

**Comment by mrigger** [comment]
Sorry, I somewhat overlooked this PR. I just noticed that some of my comments in https://github.com/sqlancer/sqlancer/pull/836 apply to this one. @ZhengLin-Li, could you perhaps have a look at them before we merge the PR?

**Comment by bajinsheng** [comment]
If this PR is deprecated by #836 , can we close it now? @ZhengLin-Li

**Comment by ZhengLin-Li** [comment]
> 

OK, close this PR，or just review and merge this PR first and then review #836

**Comment by ZhengLin-Li** [comment]
refer #836

**Pull request closed without merging**